assets/src/pages/room/hooks/usePeerState.tsx

```diff
@@ -37,7 +37,7 @@ export type PeersState = {
 
 export type Track = {
   stream?: MediaStream;
-  trackId?: string;
+  trackId: string | null;
```

Comment:
It is out of the scope of this PR but it would be nice to refactor the code so that all the current cases of `[propertyName]? : [type]` be changed to `[propertyName]: [type] | null`.